staff/acct/sorry/hibp

```diff
@@ -0,0 +1,2 @@
+Your account has been deactivated because the password associated with it
+was reused from somewhere else. Visit ocf.io/s/pwned for more information.
```

Comment:
Maybe let them know that it's due to their email address? Not sure if it's obvious

Reply:
I'd also suggest using a non /s/ shorturl since we expect those to be a little bit more permanent

Reply:
@ja5087 Nothing to do with their email address, it checks (a part of) the hash of their password against a database. See this [article](https://blog.cloudflare.com/validating-leaked-passwords-with-k-anonymity/) for why that's secure.

Reply:
Ah, I think I misunderstood HIBP then. I thought it was more we'd see if their password + email combo has been pwned, but if it's just a really big rainbow table we should let them know. Either way it's not clear what the requirement is.

Reply:
Yeah, I need to write up a page somewhere that explains how it works and why they were locked out.

a big rainbow table is good because if the password was used by someone else, it's definitely not a secure password :)

Reply:
agree about `/s/` btw